ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -182,17 +216,17 @@ export class StagePanelDef extends WidgetHost {
 
     if (!config)
       return;
-    this._size = config.size;
+    this.size = config.size;
     this._defaultSize = config.size;
     this._maxSizeSpec = config.maxSize;
     this._minSize = config.minSize;
     if (config.defaultState !== undefined) {
-      this._panelState = config.defaultState;
       this._defaultState = config.defaultState;
+      this.panelState = config.defaultState;
     }
     this._resizable = config.resizable ?? true;
     if (config.pinned !== undefined)
-      this._pinned = config.pinned;
+      this.pinned = config.pinned;
```

Comment:
Assign to field to avoid firing change events when initializing?
```suggestion
      this._pinned = config.pinned;
```